### PR TITLE
feat:created card-management page

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

File: CartManagement/cart.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <div class="badge cart-icon">
+            <a><i class="fas fa-shopping-cart"></i>
+            <div class="badge-icon">0</div></a>
+        </div>
+    </div>
+    </nav>
+    <div class="flex-container">
+        <main class="ml mt flex-container flex-columns flex-center">
+            <h2 class="brand">My Cart<span class="text-small">(2)</span></h2>
+            <div id="card-container" class="mt flex-rows">
```
Comment (surajmore98) [inline CartManagement/cart.html:34]: Try to use class instead of id

File: CartManagement/cart.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+:root{
+    --blue-hover:rgb(22, 84, 134);
+    --pink-hover:rgb(216, 144, 216);
+    --hot-pink-hover:rgb(95, 15, 66);
+}
+
+.flex-container{
+    display:flex;
+}
+.flex-rows{
```
Comment (surajmore98) [inline CartManagement/cart.css:12]: No need to specify the class for row it is default behaviour of grid

**Review by surajmore98 — APPROVED** [review]

**Review by SadiqZak — APPROVED** [review]
Looks good to merge!

File: CartManagement/cart.html (hunk 1 above)
Comment (KamiyaGaikwad) [reply CartManagement/cart.html:34]: Thank you for pointing it out. i will keep that in mind from next time

File: CartManagement/cart.css (hunk 2 above)
Comment (KamiyaGaikwad) [reply CartManagement/cart.css:12]: Okay will change that

**Pull request merged**